[INFO -- 2017-02-26 09:13:41]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-26 09:13:41]
Initialization strings:
[INFO -- 2017-02-26 09:13:41]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-26 09:13:41]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 09:13:41]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 09:13:41]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 09:13:41]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 09:13:41]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-26 09:13:41]
initSignal:4
[INFO -- 2017-02-26 09:13:41]
endSignal:8
[INFO -- 2017-02-26 09:13:41]
seqSignal:30
[INFO -- 2017-02-26 09:13:41]
faultMode:2
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 09:13:54]
pythonException: invalid frame
[INFO -- 2017-02-26 09:13:54]
Backtrace BEGIN:
#0  0x0000000000404d02 in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:148
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

143	            for (i = 0; i < BS; i++)
144	            {
145	                for (k=0; k < BS; k++) {
146	#pragma omp simd 
147	                    for (j = 0; j < BS; j++) {
148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {

Backtrace END
Memory content before bitflip:0xe00x060x000x00
Fault Mode: Random bit-flip
base address to flip value: 0x7fffffffc698
address max offset: 4
Memory content after  bitflip:0xf80x3a0xd50x7b
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: i_global
symbol filename: lud_omp.c
symbol line: 127
value: 2077571832
value address: 0x7fffffffc698
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-02-26 09:13:54]
Fault Injection Successful after 13.1298100948s
[INFO -- 2017-02-26 09:13:55]
event type: exit
[EXCEPTION -- 2017-02-26 09:13:55]
exit code: no exit code available
